Controller/ProfilerController.php

```diff
@@ -128,17 +125,6 @@ private function explainSQLServerPlatform(Connection $connection, array $query):
 
         $stmt = $connection->executeQuery($sql, $params, $query['types']);
 
-        // DBAL 2.13 "forward compatibility" BC break handling
-        if ($stmt instanceof Result) {
-            $stmt = $stmt->getIterator();
-        }
-
-        if (! $stmt instanceof PDOStatement) {
-            throw new LogicException('We need nextRowSet() functionality feature, which is not available with current DBAL driver');
```

Comment:
anyone knows why we need to call `nextRowSet` here?

Reply:
I think it is related to the way SQL server returns the statistics